tests/integration_tests/databases/api_tests.py

```diff
@@ -1111,6 +1111,7 @@ def test_get_allow_file_upload_true_csv(self):
     def mock_empty_csv_function(d, user):
         return []
 
+    @pytest.mark.skip("the excepted count is not well defined")
```

Comment:
Looked at this today in the context of your PR. The unfortunate fact is that this is a filter that depends on side effects from other tests, since it is just querying every database. When I created the test the value was consistently 0 in my local test environment, but it failed in the CI with a count of 1, which we attributed to these side effects. I think it is good to change this to 0 and not skip it.

Reply:
@villebro @amitmiran137 let me know what to do
1. change back to '0'
2. keep it as skipped
3. remove the test

Reply:
I don't think it is ok to create a test dependent on side effects of other tests .

If this test is that in stable we should remove until you can figure out a proper quality solution 

@villebro WDYT?